.eslintrc

```
@@ -21,7 +21,7 @@
         "camelcase": "error",
         "class-methods-use-this": "error",
         "complexity": ["warn", 10],
-        "consistent-return": "error",
+        // "consistent-return": "error",
```

Wäre sinnvoll, damit wir einheitliche Rückgabetypen in Funktionen erhalten
```suggestion
        "consistent-return": "error",
```